setup.py

```diff
@@ -52,13 +52,22 @@
             'pre-commit',
             'pytest>=4',
             'pytest-cov',
-            'ipython',
+            'ipython<6',
             'twine',
             'yapf',
             'prospector',
             'pylint<2; python_version<"3"',
             'pylint; python_version>="3"',
         ],
+        "docs": [
+          "Sphinx==1.8.4",
+          "Pygments==2.3.1",
+          "docutils==0.14",
+          "Jinja2==2.10.1",
+          "MarkupSafe==1.1.1",
+          "sphinx-rtd-theme==0.4.3",
+          "sphinxcontrib-contentui==0.2.2"
```

Comment:
this is for doing the folding/unfolding elements [here](https://aiida.readthedocs.io/projects/aiida-core/en/latest/install/quick_installation.html). 
If you're not using this, there is no need to add it as a dependency here.

Potentially other dependencies can be removed as wel..

Reply:
Thanks, it is removed.